rcgen/src/key_pair.rs

```diff
@@ -116,8 +118,8 @@ impl KeyPair {
 	///
 	/// Equivalent to using the [`TryFrom`] implementation.
 	#[cfg(feature = "crypto")]
-	pub fn from_der(der: &[u8]) -> Result<Self, Error> {
-		Ok(der.try_into()?)
+	pub fn from_der(der: &PrivatePkcs8KeyDer<'_>) -> Result<Self, Error> {
```

Comment:
Could you also mention in the docs that one can pass a byte slice and use `into`, maybe with an example?